pages/admin.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from "react";
+import { withRouter } from "next/router";
+import jsyaml from "js-yaml";
+import PT from "prop-types";
+
+import "../styles/admin.scss";
+
+import Layout from "../components/Layout";
+import Drawer from "../components/Drawer";
+import RenderLimberSettings from "../components/RenderLimberSettings";
+
+import RequestService from "../util/requestService";
+
+function AdminPage({ router, limberSettings }) {
+
+	// PARSE "limber/settings.yml" content as JS Object and PUT to the State BEFORE on page render
+	useEffect(() => {
+		_parseSettings(limberSettings);
+	}, []);
+
+	// DEFINE State
+	const [settings, setSettings] = useState({});
+
+	// PARSE "limber/settings.yml" content as JS Object and PUT to the State
+	async function _parseSettings(settings) {
```

Comment:
Yes, it can be there, and it makes sense.

Currently, it triggers the `_parseSettings()` function each time we click on the navigation tab, because of the URL paths: `/admin?page=pages`.
So, currently, the functionality won't change if we move it to the `_app.jsx`.

TBH, I do not like the approach of query parameters and would like to change that to the Single-Page approach (having everything rendered under the `/admin`), or Multi-Page (having different pages for each navigation item).

With the Single-Page approach, we will have a complex `pages/admin.jsx` page, all the fetch requests will be going there, etc.
With the Multi-Page approach, the functionality will be more-less distributed.